src/test/run-make-fulldeps/coverage-reports/expected_show_coverage.partial_eq.txt

```diff
@@ -2,7 +2,7 @@
     2|       |// structure of this test.
     3|       |
     4|      2|#[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord)]
-                       ^0            ^0      ^0 ^0  ^1       ^1 ^0^0
```

Comment:
Note that this fix also resolves the mystery (by removing them) of the double counters with some derived traits.

Reply:
Well, I have to take this back. After reviewing the implementation of `original_sp()`, I believe preemptively forcing the body to the root context would have changed the behavior of `original_sp()` on other source spans in a lossy way. Dropping the "extra" coverage counters here was a happy side effect of what was probably not a good change, and my new commit no longer has that side effect, but is probably more correct.